performant-software/Annotation-Studio
# Adjusted document states

#### What does this PR do?
Adjusts phases for document workflow, and adds a NAVSA-specific navbar.

Closes #51, closes #33, closes #49, closes #50 

#### How to test
- Visit the [review app](https://cove-studio-pr-69.herokuapp.com/)
- Note that there is a NAVSA logo at top right, and that navbar is black to match the one at http://dev-rc-distro.pantheonsite.io/ (corresponding changes may or may not be deployed to that site)
- Also, in the review app, navigate to `https://cove-studio-pr-69.herokuapp.com/documents`
- Note that there are now several "States" or stages for documents to move through: Editable, Annotatable, Reviewable, Publishable, and if you're an admin, Archived.
